src/Command/NormalizeCommand.php

```diff
@@ -121,19 +122,10 @@ protected function execute(Console\Input\InputInterface $input, Console\Output\O
             $composerFile = Factory::getComposerFile();
         }
 
-        try {
-            $composer = $this->factory->createComposer(
-                $io,
-                $composerFile
-            );
-        } catch (\Exception $exception) {
```

Comment:
This useless because `Factory::createComposer()` is run earlier already - we never even get here.